Gems/AWSCore/Code/Tools/ResourceMappingTool/controller/view_edit_controller.py

```diff
@@ -105,6 +105,8 @@ def _convert_and_load_into_model(self, config_file_name: str) -> None:
 
     def _create_new_config_file(self) -> None:
         configuration: Configuration = self._configuration_manager.configuration
+        self._set_default_region(configuration)
```

Comment:
There is a problem, as you are sending notification here, if following file creation failed, the notification will be overrided. (As it happens so fast, so I don't think user will have time to see the region notification)
Is it the expected behavior?

Reply:
If following file creation failed, seems like there's no need to tell them we're using us-west-2 as the default region since the file doesn't exist at all. Users are expected to restart the tool and they will still see the region error when they try to create the file again.

Also this is a temporary solution for now since Joshua believes we don't need to display this message at all once we have the files created automatically in the following ticket I'm working on.